doc/tech/approach.md

```diff
@@ -38,7 +38,7 @@ For CNTT, a strategy is needed that affords Operators the performance, flexibili
 ### 5.1.1 Networking Scope
 According to the ETSI NFV Model, Networking alongside Compute and Storage, is an integral part of the Network Function Virtualisation Infrastructure (NFVI). The general function of Networking in the NFV context is to provide the connectivity between various virtualised and non-virtualised resources required for a delivery of a Network Service. Such a connectivity may manifest itself as a virtualised network between VMs or Containers (e.g. overlay networks managed by SDN Controllers, or programmable fabrics that provide such connectivity natively) or as an integration into the Infrastructure Hardware level for offloading some of the Network Functions into this level. 
 
-The main concern of CNTT, according to its methodology, is the normalization of the integration reference points between different layers of the NFV architecture stack.  In the Networking context the primary focus is directed on the packet flow and control flow interfaces between Virtualization level (be it IaaS or CaaS type of virtualisation) and Networking Hardware resources, as well as on related integration into MANO level (Hardware/ Network Infrastructure Management/Control, Orchestration).  The CNTT scope includes hence a definition of a normalized Abstraction layer between Virtualization Layer domains and the Networking Hardware layer resource pool in a way that the implementation details of the Networking Hardware are not visible to the VNFs/CNFs, and preferably not even visible to the Virtualization layer.
+The main concern of this specification, is the normalization of the integration reference points between different layers of the NFV architecture stack.  In the Networking context the primary focus is directed on the packet flow and control flow interfaces between Virtualization level (be it IaaS or CaaS type of virtualisation) and Networking Hardware resources, as well as on related integration into MANO level (Hardware/ Network Infrastructure Management/Control, Orchestration).  The Networking Sope includes hence a definition of a normalized Abstraction layer between Virtualization Layer domains and the Networking Hardware layer resource pool in a way that the implementation details of the Networking Hardware are not visible to the VNFs/CNFs, and preferably not even visible to the Virtualization layer.
```

Comment:
Maybe we clarified this already, but what does the definition of a normalized Abstraction layer mean? Is this the design and implementation of a new, abstract API to expose networking resources to VM-s and/or containers? If yes I think this is not in the scope of CNTT, but rather should be a new project in OPNFV.

Reply:
@CsatariGergely "APIs to expose networking resources" is in the scope of upstream projects such as OpenStack (for OpenStack based IaaS).

Reply:
For me this statement sounds like something new, abstract API to expose networking resources to VM-s should be defined in CNTT, if this is something done or to be done in other open source projects, like OpenStack please add that to the text.